girder_jupyter/contents/girderfilemanager.py

```diff
@@ -261,10 +261,12 @@ def _dir_model(self, path, resource, content=True, format=None):
         if content:
             model['content'] = contents = []
             for resource in self._list_resource(resource):
-                contents.append(self._get(
-                    '%s/%s' % (path, resource['name']), resource,
-                    content=False, format=format)
-                )
+                name = resource['name']
+                if self.should_list(name) and not name.startswith('.'):
+                    contents.append(self._get(
+                        '%s/%s' % (path, resource['name']), resource,
```

Comment:
Since we already have `name = resource['name']`, we can use it in `'...'% (path, resource['name']), resource`